### PR TITLE
Overrides the github link and adds docs button

### PR DESCRIPTION
**Pull request by afzal442**
Signed-off-by: afzal442 <afzal442@gmail.com>

I have changed Github text to `Learn More` since this is already used as icon above.

### CODE REVIEW

**Comment by afzal442** [comment]
Let me know if that makes sense

cc @RainbowMango

File: data/en/banner.yml [hunk 1]
```diff
@@ -3,7 +3,11 @@ banner:
...
-  button:
+  button_gh:
     enable : true
-    label : "GitHub"
+    label : "Learn More"
     link : "https://github.com/karmada-io/karmada/"
+  button_docs:
+    enable : true
+    label : "Getting Started"
+    link : "https://github.com/karmada-io/karmada/tree/master/docs"
```
Comment (RainbowMango) [inline data/en/banner.yml:13]: Why not let the `Learn More` link to `https://github.com/karmada-io/karmada/tree/master/docs`? 
And don't need to add another button here.

Comment (afzal442) [reply data/en/banner.yml:13]: Got that. but as we have `Learn More` takes you to `karmada-io` whole repo to choose any one further and `Getting started` gives you idea to find the docs related things, it differentiates both the links. 😕

Comment (afzal442) [reply data/en/banner.yml:13]: Does that make sense? If not I can go ahead change as per your suggestion. 💯

Comment (RainbowMango) [reply data/en/banner.yml:13]: Just let the icon(GitHub icon) link to `karmada-io/karmada` repo, and the `Learn More` button link to docs would be great!

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**